testnet/ev-node/entrypoint.sh

```diff
@@ -5,44 +5,45 @@ cd /usr/bin
 
 sleep 5
 
-# Create default rollkit config if missing
+# Create default evnode config if missing
+# TODO: The --evnode.signer.path flag is not respected: https://github.com/evstack/ev-node/issues/2603
```

Comment:
Once there is a fix available for this we can have a reproducible pubkey for the sequencer in our docker network

Reply:
#153